pkg/cmd/pr/shared/survey_test.go

```diff
@@ -18,6 +18,10 @@ func (mf *metadataFetcher) RepoMetadataFetch(input api.RepoMetadataInput) (*api.
 	return mf.metadataResult, nil
 }
 
+func (mf *metadataFetcher) currentUserID() (string, error) {
+	return "1", nil
```

Comment:
Just make the tests pass. I don't know how to add a useful test for this.

This should make the options for reviewers show only `monalisa`.